src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java

```diff
@@ -104,4 +111,34 @@ public void scrapeChannelUploadPlaylist_IOException() throws IOException {
       assertThrows(IOException.class, () -> scraper.scrapeChannelUploadPlaylist(IOEXCEPTION));
   }
 
+  @Test
+  public void getPlaylistItems_nonExistentUploadId() throws IOException {
+      mockPlaylistResponse = new PlaylistItemListResponse();
+      when(mockListPlaylistItems.execute()).thenReturn(mockPlaylistResponse.setItems(null));
+      Optional<List<PlaylistItem>> actual = scraper.getPlaylistItems(UPLOAD_ID_THAT_DOES_NOT_EXIST);
+      assertEquals(false, actual.isPresent());
```

Comment:
Consider using the Hamcrest assert library as Sam mentioned [here](https://github.com/dallen68/PromoTube/pull/30#discussion_r450434236) with advantages detailed [here](https://dzone.com/articles/the-benefit-of-using-assertthat-over-other-assert).

also included in the latest commit in PR #30

Reply:
Done.